src/networktables/NetworkTable.cpp

```diff
@@ -43,6 +42,8 @@ void NetworkTable::SetTeam(int team) {
   char tmp[30];
 #ifdef _MSC_VER
   sprintf_s(tmp, "roboRIO-%d-FRC.local\n", team);
+#elif defined(__ANDROID__)
+  snprintf(tmp, 30, "roboRIO-%d-FRC.local\n",team);
```

Comment:
To get ntcore to build using the standalone android compiler, I had to use the arm settings, but change the prefix for the toolchain from ```arm-frc-linux-gnueabi-``` to ```arm-linux-androideabi-```. It would be nice to have an android.gradle and an android platform in the gradle configuration. That way we could just specify ```-Pandroid```, and perform an android build.